spec/errorSpec.js

```diff
@@ -104,6 +104,10 @@ describe('things that should fail', function() {
   });
 
   describe('native promises', function() {
+    it('should have done argument override return returned promise', function(done) {
```

Comment:
I think the description string here needs clarification - maybe:

it should time out if done argument is never called, even if promise is returned

?